src/frontend/src/common/styles/support/components/List/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { List as RawList, ListItem } from "czifui";
+import React from "react";
+
+interface Props {
+  items: React.ReactNode[];
+  ordered?: boolean;
+}
+
+const List = ({ items, ordered }: Props): JSX.Element => {
```

Comment:
would be nice if this were named something a little more descriptive?

Reply:
Thanks for the feedback! Do you mean `List` should be something more descriptive? If so it's actually a standard component name in [Material UI](https://material-ui.com/components/lists/) and IDseq's [Design System](https://chanzuckerberg.invisionapp.com/console/Design-System---Current-ckkj0euz36kkd01w6dimd8saa/ckkj0gn5p5fz401t3q88uldh5/play) 😄 !

Reply:
oh whoops i thought this was a variable that you set! nevermind!